lib/QC_Program.qpp

```diff
@@ -1713,6 +1759,22 @@ nothing Program::setGlobalVarValue(string name, auto value) {
    p->setGlobalVarValue(name->c_str(), value.refSelf(), xsink);
 }
 
+//! finds all variants of a function or class method and returns a list of the results
+/** @param function the function or class method name; may also be namespace-justified
+
+    @return a list of hashes or nothing if the name cannot be resolved; when matched, each hash element has the following keys:
+    - \c desc: a string description of the call which includes the name and the full text call signature
+    - \c params: a QoreValueList object that gives the params in a format that can be used by @ref findFunctionStatementId
+
```

Comment:
Wrong return type description (the declared return type is `list`, not `*list`). The `PROGRAM-ERROR` exception is not mentioned.